plugins/techdocs-react/api-report.md

```diff
@@ -22,6 +22,9 @@ export function createTechDocsAddonExtension<TComponentProps>(
 // @alpha (undocumented)
 export const defaultTechDocsReaderPageValue: TechDocsReaderPageValue;
 
+// @public
+export type SyncResult = 'cached' | 'updated';
```

Comment:
The naming here is pretty generic, if we think about it as part of a broader backstage project. Perhaps this move is a good opportunity to rename it to something more specific? Thinking `TechDocsStorageSyncResult` or something like that